docs/MAIN_CONTROLLER_SPECS.md

```diff
@@ -0,0 +1,34 @@
+### Specifications
+
+|Feature||
+|:------|:-----|
+|Streaming Sources | Airplay, Pandora, Spotify Connect, DLNA  |
+|||
+|Control Interfaces| Web App, REST API|
+|||
+|Dimensions        | 2U 19" rackmount unit with 300mm depth|
```

Comment:
Dimmensions and weight need to be added - L x W x H. Specify depth in the Kings units lol

Reply:
I'll have to ask for clarification on this one
I can interpret "The Kings units" to either be:
the units that kings use > units that europeans use > metric units
But I'm also aware of a legend where imperial feet were based on the length of a kings foot > imperial units

I assumed it's a saying/colloquialism that I haven't come across yet, but when searching up what "the King's units" could mean I just got information about different factions in the Total War videogame series and what units (warriors) they have access to

Reply:
The King's units are Imperial units - feet, inches, lbs, ounces, etc.